tron/bin/check_tron_jobs.py

```diff
@@ -52,6 +53,13 @@ def parse_cli():
         dest="run_interval",
         default=300,
     )
+    parser.add_argument(
+        "--skip-failed-logging-for-superregion",
+        help="Specify a superregion to skip querying stoud/stderr logs for failed jobs",
+        type=str,
+        dest="skip_failed_logging_for_superregion",
+        default=None,
+    )
```

Comment:
just curious as to what the plan for running this is: would we do some hiera/puppet shenanigans to configure the command-line for this script in the puppeted cronjob?

Reply:
I'm guessing through hiera/puppet. I'll have to see how they currently are being configured and modify it that way.